elibrary-core/source/src/controller/java/com/javafee/elibrary/core/tabbedform/TabLoanServiceEvent.java

```diff
@@ -451,4 +455,9 @@ private boolean validateLoanTableSelection() {
 		final ImportExportableJTable jTable = tabbedForm.getPanelLoanService().getLoanTable();
 		return jTable.getSelectedRow() != -1;
 	}
+	private boolean validatePenalty() {
+		double penalty = calculatePenalty();
+		if (penalty == 0) return false;
```

Comment:
Could we just return `penalty == 0` as the result? That will allow us to avoid using if statement. Please change.

Reply:
Done